File: app/models/booking.rb
# == Schema Information
#
# Table name: bookings
#
#  id         :integer          not null, primary key
#  user_id    :integer
#  flight_id  :integer
#  aisle      :integer
#  seat       :integer
#  created_at :datetime         not null
#  updated_at :datetime         not null
#

class Booking < ActiveRecord::Base

  belongs_to :flight
  belongs_to :user

end

File: app/assets/javascripts/backbone/models/booking.js
var app = app || {};

// Model.
app.Booking = Backbone.Model.extend({
  urlRoot: function () {
    return '/flights/' + this.get('id') + '/bookings';
  },
  defaults: {
    user_id: 1,
    flight_id: 1,
    aisle: 1,
    seat: 1
  }
});